packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -146,12 +159,30 @@ const setupDriver = async () => {
 		}
 	}
 
-	if ( ! isLocalEnvironment() ) {
-		desiredCaps.name = `Gutenberg Editor Tests[${ rnPlatform }]-${ branch }`;
-		desiredCaps.tags = [ 'Gutenberg', branch ];
-	}
+	const sauceOptionsConfig = ! isLocalEnvironment()
+		? {
+				'sauce:options': {
+					...sauceOptions,
+					name: `Gutenberg Editor Tests[${ rnPlatform }]-${ branch }`,
+					tags: [ 'Gutenberg', branch ],
+					idleTimeout: 180,
+				},
+		  }
+		: {};
+	const serverConfig = isLocalEnvironment()
+		? serverConfigs.local
+		: serverConfigs.sauce;
 
-	await driver.init( desiredCaps );
+	const driver = await remote( {
+		...serverConfig,
+		waitforTimeout: 60000,
+		logLevel: 'error',
+		capabilities: {
+			platformName: PLATFORM_NAME,
+			...prefixKeysWithAppium( desiredCaps ),
```

Comment:
At first I was using `appium:options` to pass all of these capabilities, but then I found an issue on SauceLabs when it was trying to trigger real devices instead of simulators. It turns out this option of using `appium:options` is only available for Real devices ([docs](https://docs.saucelabs.com/dev/test-configuration-options/#appiumoptions)).

I'm open to other ideas on how to dynamically rename all capabilities to have the `appium:` prefix.